hazelcast/src/test/java/com/hazelcast/client/tpc/ClientTpcTest.java

```diff
@@ -78,39 +80,50 @@ public void cleanup() {
 
     @Test
     public void testClientConnectsAllTpcPorts() {
```

Comment:
should not this test fail? Did you try running this test class and make sure all pass? To make this test pass, I think you need to set tpc connection count to event loop count.